rpcs3/rpcs3qt/gui_application.cpp

```diff
@@ -135,10 +135,19 @@ bool gui_application::Init()
 	// Create connects to propagate events throughout Gui.
 	InitializeConnects();
 
+	bool set_dark_theme = false;
+
 	if (m_gui_settings->GetValue(gui::ib_show_welcome).toBool())
 	{
 		welcome_dialog* welcome = new welcome_dialog(m_gui_settings, false);
```

Comment:
You have to delete this pointer before it goes out of scope, this block of code is memory leaking.

Reply:
It's not, QT takes care of it.

Reply:
Woah, didn't know that. Sorry.